packages/design-system/src/components/card/CardProduct.tsx

```diff
@@ -0,0 +1,83 @@
+import { SvgLikeFill } from '../../icons/fill/components/LikeFill';
+import { SvgLikeOutline } from '../../icons/fill/components/LikeOutline';
+import { SvgStar } from '../../icons/fill/components/Star';
+import { Badge } from '../badge';
+
+interface CardProductProps {
+  rank: number;
+  brand: string;
+  title: string;
+  description: string;
+  productId: number;
+  isLiked: boolean;
+  rating: number;
+  reviewCount: number;
+  likeCount: number;
+  imageUrl?: string;
+  handleLikeToggle?: (productId: number, isLiked: boolean) => void;
+  handleCardClick?: (productId: number) => void;
+}
+
+export default function CardProduct({
+  rank,
+  brand,
+  title,
+  description,
+  productId,
+  isLiked,
+  rating,
+  reviewCount,
+  imageUrl,
+  handleCardClick,
+  handleLikeToggle,
+}: CardProductProps) {
+  const handleLikeClick = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    handleLikeToggle?.(productId, isLiked);
+  };
+
+  return (
+    <article
+      className="flex w-[16.5rem] cursor-pointer flex-col"
+      onClick={() => handleCardClick?.(productId)}
+    >
+      <div className="relative border-[0.0625rem] border-gray-200">
+        {imageUrl ? (
+          <img
+            className="h-[16.5rem] w-[16.5rem] object-cover"
+            src={imageUrl}
+            alt={title}
+          />
+        ) : (
+          <div className="flex h-[16.5rem] w-[16.5rem] items-center justify-center">
+            상품 이미지 준비중
+          </div>
+        )}
+        <Badge rank={rank} />
+      </div>
+      <div className="flex h-[2.75rem] items-center justify-between border-b-[0.0625rem] border-dashed border-pink-500">
+        <p className="text-jp-body1 font-[700]">{brand}</p>
+        <button onClick={handleLikeClick}>
+          {isLiked ? (
+            <SvgLikeFill size={24} fill="#ff487f" />
```

Comment:
```suggestion
            <SvgLikeFill size={24} className="fill-pink-500" />
```
이 색 맞는지는 모르겟는데 추측해봄... className 사용해서 토큰으로 적용합쉬다~

Reply:
이거 디자인 토큰이 적용이 안돼서 색상값으로 적용한거긴 한데 다른 부분 수정하면서 다시 확인해볼게요!